desktop/apps/editorial_features/components/gucci/components/series/series_footer.jsx

```diff
@@ -0,0 +1,74 @@
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { Col, Row } from 'react-styled-flexboxgrid'
+import { pMedia } from '@artsy/reaction-force/dist/Components/Helpers'
+import { Fonts } from '@artsy/reaction-force/dist/Components/Publishing/Fonts'
+import { Text } from '@artsy/reaction-force/dist/Components/Publishing'
+import { PartnerFooter } from '../partner/partner_footer.jsx'
+
+export const SeriesFooter = (props) => {
+  const { curation } = props
+  const logoSrc = curation.partner_logo_footer || curation.partner_logo_primary
+
+  return (
+    <SeriesFooterContainer>
+      <Row className='SeriesFooter'>
+        <Col sm={4} className='col col--first'>
+          <Title>About the Series</Title>
+          <PartnerFooter
+            logo={logoSrc}
+            url={curation.partner_link_url}
+          />
+        </Col>
+        <Col sm={7} className='col col--last'>
+          <Text html={curation.about} />
+          <PartnerFooter
+            logo={logoSrc}
+            url={curation.partner_link_url}
+          />
```

Comment:
Are there supposed to be two PartnerFooters?

Reply:
Yeah it is to deal with position changes in desktop v mobile -- ideally our container would know which one it is, but maybe I can get that into a later PR